README.md

```diff
@@ -1,8 +1,62 @@
-Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Jasmeet Brar . Jarvis Consulting
 
+I recently graduated from University of Toronto with an Honours Bachelor of Science (Computer Science Specialist and Math Minor), and I intend to venture into software development and data engineering. With my strong background in mathematics, it became natural that I am centered around problem-solving, and being able to construct elegant solutions that are extendable, scalable, and maintainable. In terms of academics, I have studied operating systems, algorithms, computability, web development, parallel programming, scalable computing, machine learning, deep learning, and much more. I have experience in projects involving compilers, web programming, scripting, and data visualization. As for work experience, I had worked as a software developer in helping a firm migrate its frontend services, and I currently work at Jarvis as a data engineer, where I work on projects that involve constructing some pipelines to deliver data in a format that would allow for further analysis. Altogether, they make up all the skills and experience that I want to carry forward towards new opportunities. New opportunities that will make use of my abilities and allow me to further learn new technologies. I am passionate about this journey that I'm on, and I can't wait to see what's up ahead.
```

Comment:
Check your spelling and grammar through https://www.grammarly.com/

Reply:
Done.